CONTRIBUTING.rst

```diff
@@ -30,6 +36,13 @@ Implement Features
 Look through the GitHub issues for features. Anything tagged with "feature"
 is open to whoever wants to implement it.
 
+
+Improvement requests
+~~~~~~~~~~~~~~~~~~~~
+
+If you see some of the things could be done in a better way, feel free to submit a issue with a improvement suggestion.
+In suggestion you might include motivation for improvement and why new solution would be improvement.
```

Comment:
Wording suggestion:

Please motivate your suggestion by illustrating a problem it solves.